_data/utilities.yml

```diff
@@ -240,6 +240,16 @@ websites:
       img: tmobile.png
       tfa: No
 
+    - name: Telia
```

Comment:
Since the merge in '08 it's now called TeliaSonera however they operate under the brands Telia and Halebop in Sweden.
I've thought a bit on whether we should change the name in this PR to TeliaSonera however since Telia is their brand name I suppose we should keep it as it is.
Along with the merge they bought up companies in some countries and bought monopolies in others.

As this PR is intented for the Swedish subsection of the Telia Company I recommend you up "(Sweden)" after the name to reflect this.

So please change the value of the name tag to `Telia (Sweden)`

Many thanks!
//Carl
<br>
<hr>

_Sources:_
* _[Wikipedia](https://en.wikipedia.org/wiki/Telia_Company)_
* _[teliacompany.com](http://www.teliacompany.com/en)_